dpnp/backend/CMakeLists.txt

```diff
@@ -186,6 +187,13 @@ endif()
 target_include_directories(dpnp_backend_c PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
 target_include_directories(dpnp_backend_c PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/src)
 
+# -----------------------------------------------------------------------------------------------
+# Testing logic...
+# -----------------------------------------------------------------------------------------------
+if(DPNP_BACKEND_TESTS)
```

Comment:
What will happen is nobody provides this option at compile time?
add `option(DPNP_` with default `FALSE` as above